spec/support/coronavirus_feature_steps.rb

```diff
@@ -344,16 +350,48 @@ def then_a_new_timeline_entry_is_created
 # Editing timeline entries
 
 def when_i_visit_the_edit_timeline_entry_page
-  visit "/coronavirus/landing/timeline_entries/#{@timeline_entry.id}/edit"
+  visit "/coronavirus/landing/timeline_entries/#{@timeline_entry_one.id}/edit"
 end
 
 def and_i_see_the_existing_timeline_entry_data
-  expect(page).to have_selector("input[value='#{@timeline_entry.heading}']")
-  expect(page).to have_content(@timeline_entry.content)
+  expect(page).to have_selector("input[value='#{@timeline_entry_one.heading}']")
+  expect(page).to have_content(@timeline_entry_one.content)
 end
 
 def then_the_timeline_entry_is_updated
-  expect(@timeline_entry.reload.content).to eq("##Form content")
+  expect(@timeline_entry_one.reload.content).to eq("##Form content")
+end
+
+# Reordering timeline entries
+
+def when_i_visit_the_reorder_timeline_entries_page
+  visit "/coronavirus/landing/timeline_entries/reorder"
+end
+
+def then_i_see_the_timeline_entries_in_order
+  element = find("#step-0").find(".step-by-step-reorder__step-title")
+  expect(element).to have_content @timeline_entry_one.heading
+
+  element = find("#step-1").find(".step-by-step-reorder__step-title")
+  expect(element).to have_content @timeline_entry_two.heading
+end
+
+def when_i_move_timeline_entry_one_down
+  raw_content = File.read(Rails.root.join("spec/fixtures/coronavirus_landing_page.yml"))
+  stub_request(:get, /#{@coronavirus_page.raw_content_url}\?cache-bust=\d+/)
+    .to_return(status: 200, body: raw_content)
+
+  find("#step-0").find(".js-order-controls").find(".js-down").click
+  click_button "Save"
+end
+
+def then_i_see_timeline_entries_updated_message
+  expect(page).to have_content "Timeline entries were successfully reordered."
+end
+
+def and_i_see_the_timeline_entries_have_changed_order
+  expect(page).to have_css(".covid-manage-page__summary-list--divider .gem-c-summary-list .govuk-summary-list__row:nth-child(1)", text: @timeline_entry_two.heading)
+  expect(page).to have_css(".covid-manage-page__summary-list--divider .gem-c-summary-list .govuk-summary-list__row:nth-child(2)", text: @timeline_entry_one.heading)
```

Comment:
This too could be a single assertion of `expect(page).to have_content(/#{@timeline_entry_two.heading}.*#{@timeline_entry_one.heading}/, normalize_ws: true)` which might be a bit nicer than needing to dig deep into the css selectors.